### PR TITLE
Add initial CI pipeline

### PR DESCRIPTION
**Pull request by kylelaker-ccpo**

### CODE REVIEW

**Comment by kylelaker-ccpo** [comment]
@jasonburkert-ccpo some of these are the same as your PR #40. My preference would be to merge them here to simplify your PR but we can also hold this until your PR is done and I could rebase. But getting this PR in sooner rather than later would allow for both your PR and #44 to benefit from these changes.

File: applications/portfolioDrafts/package-lock.json [hunk 1]
```diff
@@ -1,2038 +1,8 @@
 {
 	"name": "portfoliodrafts",
 	"version": "0.1.0",
-	"lockfileVersion": 2,
+	"lockfileVersion": 1,
```
Comment (zachclark-ccpo) [inline applications/portfolioDrafts/package-lock.json:4]: Is this correct? I thought we were using npm `7.19.1`?

Comment (zachclark-ccpo) [reply applications/portfolioDrafts/package-lock.json:4]: Read the comment in atat-platform, this is the correct lockfileVersion.

**Review by zachclark-ccpo — APPROVED** [review]
This checks out, good work.

**Pull request merged**